docs/contributing/development.md

```
@@ -33,15 +33,18 @@ The project has two main parts: the OpenSCAD 3D models and the Python scripts fo
+  uv venv
   source .venv/bin/activate  # On Windows, use `.venv\Scripts\activate`
   ```
-- **Dependencies**: Install the required dependencies using `pip`:
+- **Dependencies**: Install the required dependencies using `uv`:
   ```bash
-  python -m pip install --upgrade pip
-  pip install -r requirements.txt
+  uv pip install -r requirements.txt
+  ```
+- **Updating requirements.txt**: When modifying `requirements.in`, recompile the lock file:
+  ```bash
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)